### PR TITLE
Fix: remove window dependent styles

### LINKED ISSUE #422: Landing Page is not displayed properly on the moment its opened
Its around a second for desktop and a bit longer for mobile, but at least its not happening on every refresh

screenshot 2019-02-27 at 13 49 18

### PR DESCRIPTION
**Pull request by KarinaDavtyan**
Summary: remove HOC `withWindowSize` and instead of that use media queries

Closes #422

### CODE REVIEW

File: apps/landingPage/components/Info/ResponsiveText.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    display: none;
+  }
+`;
+
+const Desktop = styled.div`
+  display: none;
+  @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) {
+    display: flex;
+  }
+`;
```
Comment (FilipMessa) [inline apps/landingPage/components/Info/ResponsiveText.js]: These `Mobile` and `Desktop` styled components look the same as in the `ButtonInfo.js` so probably we can extract them and reuse. 🤔

Comment (KarinaDavtyan) [reply apps/landingPage/components/Info/ResponsiveText.js]: Should I extract them together with `Tablet` into `landingPage/components/styles/index.js` ?

Comment (FilipMessa) [reply apps/landingPage/components/Info/ResponsiveText.js]: 🤔 I didn't realize that. So, in that case, it is OK.

File: apps/landingPage/components/Info/ResponsiveText.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          {text}
+        </Text>
+      </Mobile>
+      <Desktop>
+        <Text size="large" align="center">
+          {text}
+        </Text>
+      </Desktop>
+    </WrapperText>
+  );
```
Comment (FilipMessa) [inline apps/landingPage/components/Info/ResponsiveText.js]: With this approach there will be the same text two times in HTML, shouldn't be better do it with nested styles? 
something like:
```
 @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) {
	  > p {
	      text-align: center;
	    }
```

Comment (KarinaDavtyan) [reply apps/landingPage/components/Info/ResponsiveText.js]: Much better like that!

**Pull request merged**